ci-operator/step-registry/ipi/conf/azure/bootdiagnostics/ipi-conf-azure-bootdiagnostics-commands.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "controlPlane boot diagnostics type: ${AZURE_CONTROL_PLANE_BOOT_DIAGNOSTICS_TYPE}"
+echo "Compute boot diagnostics type: ${AZURE_COMPUTE_BOOT_DIAGNOSTICS_TYPE}"
+echo "DefaultMachinePlatform boot diagnostics type: ${AZURE_DEFAULT_MACHINE_BOOT_DIAGNOSTICS_TYPE}"
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+if [[ -f "${SHARED_DIR}/azure_storage_account_name" ]] && [[ -f "${SHARED_DIR}/resourcegroup_sa" ]]; then
```

Comment:
Suggest to split the if block into two if blocks, though generally both of them are used together.
```
if [[ -f "${SHARED_DIR}/azure_storage_account_name" ]]; then
...
fi
if [[ -f "${SHARED_DIR}/resourcegroup_sa" ]]; then
...
fi
```

Reply:
updated.